Content.Server/Interaction/InteractionPopupSystem.cs

```diff
@@ -61,10 +61,13 @@ private void OnInteractHand(EntityUid uid, InteractionPopupComponent component,
         else
             _popupSystem.PopupEntity(msg, uid, Filter.Entities(args.User)); //play only for the initiating entity.
 
-        if (component.SoundPerceivedByOthers)
-            SoundSystem.Play(Filter.Pvs(args.Target), sfx, args.Target); //play for everyone in range
-        else
-            SoundSystem.Play(Filter.Entities(args.User, args.Target), sfx, args.Target); //play only for the initiating entity and its target.
+        if (sfx != "") //not all cases will have sound.
```

Comment:
```suggestion
        if (sfx != string.Empty) //not all cases will have sound.
```

or make it nullable, that would probably be preferable

Reply:
Made it nullable on latest commit. Would you please make sure I've done so correctly? I've tested on local and it still runs fine.